src/main/java/aniela/remanent/position/abstraction/PositionInterface.java

```diff
@@ -0,0 +1,47 @@
+package aniela.remanent.position.abstraction;
+
+import aniela.remanent.type.ReportType;
+
+public interface PositionInterface {
```

Comment:
slaba nazwa PositionInterface
jak juz cos to samo Position

Reply:
OK. Racja